pkg/pillar/utils/volumeutils.go

```diff
@@ -26,6 +27,17 @@ func GetVolumeSize(log *base.LogObject, name string) (uint64, uint64, string, bo
 		size, err := diskmetrics.SizeFromDir(log, name)
 		return size, size, "CONTAINER", false, err
 	}
+	if info.Mode()&os.ModeDevice != 0 {
+		//Assume this is zfs device
```

Comment:
Why this can not be an ext4 volume?

Reply:
That's because currently the pathname is either a directory (the container rootfs) or a file (with a qcow2 or raw).
Only with ZFS do we create them as devices.